script.js

```diff
@@ -187,28 +213,43 @@ const stopTimer = () => {
 // new game button fuction------------------------------------------------
 newGameButtons.forEach((newGameButton) => {
   newGameButton.addEventListener("click", () => {
+    resetGame()
     alertOverlay.style.display = "none";
     alertContainer.style.display = "none";
     // reset cards
-    cardContainer.innerHTML = "";
-    getData()
-
-    // reset timer
-    clickCounter = 0;
-    seconds.textContent = "00:00";
-    stopTimer();
-
-    // reset movements
-    movementsCounter = 0;
-    movementsContainer.textContent = 0;
+    if(isEasyGame){
```

Comment:
This is small and simple enough to become a ternary operation 💡

Reply:
Sure